src/claude-session-utility.ts

```
@@ -277,7 +296,7 @@ end tell`);
     const candidates = allSessions.filter(s => s.project === cwd);
     const encodedProject = cwd.replace(/[^a-zA-Z0-9-]/g, '-');
     await Promise.all(candidates.map(async (session) => {
-      const jsonlPath = path.join(claudeDir, encodedProject, `${session.sessionId}.jsonl`);
+      const jsonlPath = path.join(getProjectsDir(session.accountDir), encodedProject, `${session.sessionId}.jsonl`);
```

ESLint

[error] Replace `const·jsonlPath·=·path.join(getProjectsDir(session.accountDir),·encodedProject,·`${session.sessionId}.jsonl`` with `··const·jsonlPath·=·path.join(⏎··········getProjectsDir(session.accountDir),⏎··········encodedProject,⏎··········`${session.sessionId}.jsonl`,⏎········`

(prettier/prettier)